lua/lsp/ts-fmt-lint.lua

```diff
@@ -2,23 +2,41 @@
 -- You can look for project scope Prettier and Eslint with e.g. vim.fn.glob("node_modules/.bin/prettier") etc. If it is not found revert to global Prettier where needed.
 local M = {}
 
+vim.cmd "let proj = FindRootDirectory()"
```

Comment:
can you move these lines to `M.setup` function  (L5 to L15)

Reply:
One moment. I made a incorrect git rebase...will fix soon.

Reply:
Lines have now been moved to `M.setup`. Note also, that I added `formattingSupported` to disable the formatting support on efm depending on `eslint_d`